src/drive.py

```diff
@@ -1,10 +1,79 @@
+# import db
+from Google import connect_to_drive
+import pandas as pd
 import db
 
+
+def get_folder_contents(service, id):
+
+    query = f"parents = '{id}'"
```

Comment:
this just work for some reason.
```suggestion
    query = f"'{id}' in parents"
```

Reply:
updated